CONTRIBUTING.adoc

```diff
@@ -246,11 +246,11 @@ Make sure that all tests pass prior to submitting your pull request.
 
 == Mention your pull request on the associated GitHub issue
 
-Add a comment to the associated JIRA issue(s) linking to your new pull request.
+Add a comment to the associated GitHub issue(s) linking to your new pull request.
 
-== Provide a Link to the JIRA issue in the Associated Pull Request
+== Provide a Link to the GitHub issue in the Associated Pull Request
 
-Add a JIRA issue link to your first commit comment of the pull request on the last line, so your commit message
+Add a GitHub issue link to your first commit comment of the pull request on the last line, so your commit message
```

Comment:
Please, change this to reflect reality: "on the second line".
Also would be great to mention a trick with key words to close an issue automatically: https://help.github.com/en/enterprise/2.16/user/github/managing-your-work-on-github/closing-issues-using-keywords